### PR TITLE
Add support to create private issues.

### PR DESCRIPTION
**Pull request by shreyaspapi**
- [x] Gitlab - Private issues are known as confidential issues
- [x] Github - Does not support private/confidential issues. (raised an error here)
- [x] Pagure

### CODE REVIEW

**Comment by shreyaspapi** [comment]
This was supposed to be a draft PR. I am making more changes.

File: tests/integration/test_gitlab.py [hunk 1]
```diff
@@ -246,19 +246,38 @@ def test_create_issue(self):
...
+            assert issue_label.name == label
+
+    def test_create_private_issue(self):
+        """
+        see class comment in case of fail
+        """
+        issue_title = f"New Confidential Issue {self.random_str}"
+        issue_desc = f"Description for issue {self.random_str}"
+        issue = self.project.create_issue(
+            title=issue_title, body=issue_desc, private=True
```
Comment (lachmanfrantisek) [inline tests/integration/test_gitlab.py:268]: Can you please create also tests for other forges that will check that we are correctly raising an error?

Comment (shreyaspapi) [reply tests/integration/test_gitlab.py:275]: Yes, added it [here](https://github.com/packit-service/ogr/pull/441/files#diff-6f042b301fdcde9c4256f0b7bfe15798R411-R412 )for Github, Already added tests for pagure for private issue.

File: tests/integration/test_github.py [hunk 2]
```diff
@@ -397,6 +397,20 @@ def test_issue_list(self):
...
+        labels = ["label1", "label2"]
+        project = self.service.get_project(namespace="shreyaspapi", repo="test")
+        issue = project.create_issue(title=title, body=description, labels=labels)
+        assert issue.title == title
+        assert issue.description == description
+        for issue_label, label in zip(issue.labels, labels):
+            assert issue_label.name == label
+
+        with self.assertRaises(NotImplementedError):
+            project.create_issue(title=title, body=description, private=True)
```
Comment (shreyaspapi) [inline tests/integration/test_github.py:412]: @lachmanfrantisek Added here!

**Comment by shreyaspapi** [comment]
@lachmanfrantisek Made a new commit and tests passed!

File: ogr/abstract.py [hunk 3]
```diff
@@ -190,7 +195,6 @@ def create(project: Any, title: str, body: str) -> "Issue":
     def get(project: Any, id: int) -> "Issue":
         """
         Get issue.
-
```
Comment (lachmanfrantisek) [inline ogr/abstract.py:193]: Please, don't remove those lines...

**Pull request merged**